CRM/Contribute/Tokens.php

```diff
@@ -46,4 +48,26 @@ public function getCurrencyFieldName() {
     return ['currency'];
   }
 
+  /**
+   * Get Related Entity tokens.
+   *
+   * @return array[]
+   */
+  protected function getRelatedTokens(): array {
+    $tokens = [];
+    $hiddenTokens = ['modified_date', 'create_date', 'trxn_id', 'invoice_id', 'is_test', 'payment_token_id', 'payment_processor_id', 'payment_instrument_id', 'cycle_day', 'installments', 'processor_id', 'next_sched_contribution_date', 'failure_count', 'failure_retry_date', 'auto_renew', 'is_email_receipt', 'contribution_status_id'];
```

Comment:
I wonder about next_sched_contribution_date - that feels like something people might use - but non-blocking